pkg/reconciler/workload/placement/placement_reconcile_scheduling_test.go

```diff
@@ -93,6 +95,22 @@ func TestSchedulingReconcile(t *testing.T) {
 				workloadv1alpha1.InternalSyncTargetPlacementAnnotationKey: "aPkhvUbGK0xoZIjMnM2pA0AuV1g7i4tBwxu5m4",
 			},
 		},
+		{
```

Comment:
Do we want to add another test when the only SyncTarget in the Location is incompatible (or simply does not have the API) ? This should fail, right ?